pal/src/tlsio_arduino.c

```diff
@@ -514,23 +533,29 @@ static void tlsio_arduino_dowork(CONCRETE_IO_HANDLE tls_io)
         switch (tls_io_instance->tlsio_state)
         {
         case TLSIO_STATE_CLOSED:
+	    LogInfo("dowork TLSIO_STATE_CLOSED");
             /* Codes_SRS_TLSIO_30_075: [ If the adapter is in TLSIO_STATE_EXT_CLOSED then  tlsio_dowork  shall do nothing. ]*/
             // Waiting to be opened, nothing to do
             break;
         case TLSIO_STATE_OPENING_WAITING_DNS:
+            LogInfo("dowork TLSIO_STATE_OPENING_WAITING_DNS");
             dowork_poll_dns(tls_io_instance);
             break;
         case TLSIO_STATE_OPENING_WAITING_SOCKET:
+            LogInfo("dowork TLSIO_STATE_OPENING_WAITING_SOCKET");
             dowork_poll_socket(tls_io_instance);
             break;
         case TLSIO_STATE_OPENING_WAITING_SSL:
+            LogInfo("dowork TLSIO_STATE_OPENING_WAITING_SSL");
             dowork_poll_open_ssl(tls_io_instance);
             break;
         case TLSIO_STATE_OPEN:
+            // LogInfo("dowork TLSIO_STATE_OPEN");
```

Comment:
any reason this is still here after being commented out?

Reply:
Yeah, for if you want to be spammed by this status *shrugs*

Reply:
It should be removed then

Reply:
Removed.